src/commands/hld/init.test.ts

```diff
@@ -0,0 +1,43 @@
+import fs from "fs";
+import os from "os";
+import path from "path";
+import uuid from "uuid/v4";
+import {
+  disableVerboseLogging,
+  enableVerboseLogging,
+  logger
+} from "../../logger";
+import { initialize } from "./init";
+
+beforeAll(() => {
+  enableVerboseLogging();
+});
+
+afterAll(() => {
+  disableVerboseLogging();
+});
+
+describe("Initializing an HLD Repository.", () => {
+  test("New directory is created under root directory with required service files.", async () => {
+    // Create random directory to initialize
```

Comment:
recommend using `mockfs` when testing directory/file creation.

Here's an example on how to use `mockfs` https://github.com/CatalystCode/spk/blob/57bc476ced54b586e6e3b77390ba713d8f5f0b53/src/lib/fileutils.test.ts